examples/settings.gradle

```diff
@@ -12,6 +12,6 @@ include 'newsreaderExample'
 include 'rxJavaExample'
 include 'secureTokenAndroidKeyStore'
 include 'threadExample'
-include 'unitTestExample'
+// include 'unitTestExample' Disable project because fixing it requires AndroidX
```

Comment:
It's hard to keep up with all these TODO/disabled parts, can we centralize these changes in one issue for coordination?

Reply:
Not really. It is a huge mess I agree. I'm trying to make the PR's as isolated as possible and adding any tasks to https://github.com/realm/realm-java/pull/6756 but it is far from ideal.